syncer.py

```diff
@@ -231,6 +232,46 @@ async def sync_intraday(self, a):
         await series.kv.update(sync_query=a["sync_query"].isoformat())
         a["sync_query"] = a["sync_query"] + timedelta(days=1)
 
+    async def sync_body_data(self, body_data_config):
+        # we fetch multiple body data series with only a single API request.
+        # As a result, for the sync_query, we just use the sync query of the weight
+        sync_query = body_data_config["weight"]["sync_query"]
+        curdate = datetime.now(tz=self.timezone).date()
+        if curdate < sync_query:
+            # Skip if already finished sync
+            return
+        query_end = sync_query + timedelta(days=30)  # Query by 30 days  # todo how many days are best?
```

Comment:
I'm not sure what duration would be suitable here... A large time range like 30 days would significantly speed up the "initial sync" and reduce the risk of hitting the rate limits. On the other hand, once the initial sync is done, 30 days is pretty useless, considering that we're syncing hourly.

Reply:
> Notice that once the initial sync is done, we won't actually be getting all 30 days, since the code right below this line will limit the query range to the current date as endpoint

you're obviously right :+1: 

I also quickly tried to find the largest supported time range, but the doc doesn't mention it. :shrug: